coderd/notifications/testdata/rendered-templates/TemplateTemplateDeleted.json.golden

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Note to self: This serialisation of the payload with rendered body and title is useful. It proves that all the right things are passed to the dispatcher. However, it doesn't prove that the smtp dispatcher renders the actions correctly. I should consider adding golden tests to the smtp dispatcher